SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Source.cs

```diff
@@ -234,11 +424,27 @@ public static void ForceCurrent(this DCPowerSessionsBundle sessionsBundle, IDict
             {
                 var perPinSettings = settings[sitePinInfo.PinName];
                 perPinSettings.OutputFunction = DCPowerSourceOutputFunction.DCCurrent;
-                perPinSettings.LimitSymmetry = DCPowerComplianceLimitSymmetry.Symmetric;
                 sessionInfo.Force(perPinSettings, sitePinInfo.IndividualChannelString, waitForSourceCompletion);
             });
         }
 
+        /// <summary>
+        /// Forces current and specifies voltage limit.
```

Comment:
Suggest changing to: "Forces current using specified pin- and site-unique source settings."